gcm/README.md

```diff
@@ -0,0 +1,98 @@
+# Parse SDK Android GCM
+GCM support for Parse Android apps
+
+## Deprecated
+Please note that GCM is deprecated in favor of FCM. This module exists as a backwards compatible solution for projects already using GCM. New apps should instead use FCM.
+
+## Setup
+
+### Installation
+
+Add dependency to the application level `build.gradle` file.
+
+```groovy
+dependencies {
+    implementation 'com.parse:parse-android:latest.version.here'
+    implementation 'com.parse:parse-android-gcm:latest.version.here'
+
+    implementation "com.google.android.gms:play-services-gcm:latest.version.here"
+}
+```
+You will then need to register some things in your manifest, firstly, the GCM sender ID:
+```xml
+<meta-data
+    android:name="com.parse.push.gcm_sender_id"
+    android:value="id:YOUR_SENDER_ID_HERE" />
+```
+The sender ID should be all numbers. Make sure you are keeping the `id:` in the front
+
+Next:
+```xml
+<receiver
+    android:name="com.google.android.gms.gcm.GcmReceiver"
+    android:exported="true"
+    android:permission="com.google.android.c2dm.permission.SEND" >
+    <intent-filter>
+        <action android:name="com.google.android.c2dm.intent.RECEIVE" />
+        <category android:name="com.your.package.name.here" />
```

Comment:
You should be able to use so you don't need to worry about mistyping.  Gradle swaps in the correct package name.

```
<!-- ${packageName} is substituted for the package name declared above.  -->
  <permission android:protectionLevel="signature" android:name="${packageName}.permission.C2D_MESSAGE" />
  <uses-permission android:name="${packageName}.permission.C2D_MESSAGE" />```